web/app/api/search.py

```diff
@@ -32,11 +32,29 @@ def search_column(body: ColumnSearch):
         if not model_view.column_list:
             continue
 
-        if body.col_name in model_view.column_list:
+        # get current model name from url
+        model_name = body.current_href.split("/")[4]
```

Comment:
I don't like this line. 
1. What would happen if we don't have 5th element in array (most likely server will crash due to KeyError)?
2. we need current ref only for getting arbitrary model name? if so, I think we shouldn't send whole href, just model name as optional string, then this code would be simpler and more reliable